lib/classification_lifecycle.rb

```diff
@@ -37,6 +37,7 @@ def process_project_preference
       if first_classifcation = upp.email_communication.nil?
         Project.increment_counter :classifiers_count, project.id
         upp.email_communication = user.project_email_communication
+        upp.user.update( {project_id: project.id} )
```

Comment:
Should a user's project_id not have to reference an actual project? If not, would it make more sense to remove the FK constraint?